mathsys/dev/core/data/_expression.rs

```
@@ -0,0 +1,52 @@
+//^
+//^ HEAD
+//^
+
+//> HEAD -> CROSS-SCOPE TRAIT
+use crate::class::Class;
+use crate::object::Object;
+use crate::runtime::Runtime;
+use crate::tip::Tip;
+use crate::group::Group;
```

Clippy (1.91.1)

[warning] unused import

(warning)